Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/bugreporting/util/Converters.kt

```diff
@@ -0,0 +1,26 @@
+package de.rki.coronawarnapp.bugreporting.util
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import org.joda.time.Instant
+
+class Converters {
+
+    private val gson = Gson()
```

Comment:
@d4rken introduced an injectable GsonBuilder the other day.

Reply:
If I remember right, my PR was published first. Cannot predict the future :-)
But thx for info.
adjusted.